packages/kotti-ui/source/kotti-field/KtField.vue

```diff
@@ -99,6 +103,7 @@ import FieldHelpText from './components/FieldHelpText.vue'
 import { KottiField } from './types'
 
 export default defineComponent<{
+	containerPadding: string | null
```

Comment:
heavily opposing this. 
> Most of our goal when working on the new fields/form was to reduce the size of the API, keeping it short and concise (having exactly the amount of props it needs to not compromise API simplicity over flexibility).

There is a tendency to misuse an API that is huge, since the proper intended usage can get lost in translation.

Another thing is, this leaves room for inconsitent UI already. something the designers are heavily opposed to. 
> one middle ground solution I see is having reactive button sizes with a minimum width depending on the container size

OR: (discuss with designers) 
hiding the maximum when the field container is small (and introducing it as a tippy or as help text)

OR even to the label of the field - think about
```
amount (/100):
[ KT FIELD NUMBER ] 
```

Reply:
you also have the ability to `hideMaximumButtons` so you can have a media-query in the user app that hides the buttons entirely

but I don't this as the only solution that will attend to ur current problem, keep the API at minimum and provided a consistent UI on the long run

Reply:
Yeah, personally also not too happy with this, which is one of the main reasons why I decided to split-out the non-controversial changes. So that we can already merge those simple fixes without being blocked by this discussion etc.

Reply:
@carsoli the label is also problematic as it’s translated — and thus may take up _a lot_ more space for other languages with higher redundancy

_\*cough\* German_

Reply:
@carsoli what do you think about using a negative margin on the buttons instead? That’d require no additional `KtField` API and could be done entirely in CSS

Reply:
It’d also prevent this misalignment issue as the buttons wouldn’t even be shown in that case — thus they couldn’t add negative margins: 
![image](https://user-images.githubusercontent.com/1133858/115036297-acd42280-9ecd-11eb-9f29-5c8d313b9949.png)